transport/p2p/src/swarm.rs

```diff
@@ -49,13 +49,9 @@ async fn build_p2p_network(
 ) -> Result<libp2p::Swarm<HoprNetworkBehavior>> {
     let tcp_upgrade = libp2p::core::upgrade::SelectUpgrade::new(
         {
-            let num_streams = std::env::var("HOPR_INTERNAL_LIBP2P_YAMUX_MAX_NUM_STREAMS")
```

Comment:
Since the `HOPR_INTERNAL_LIBP2P_YAMUX_MAX_NUM_STREAMS` env variable is removed from the code, it should be removed from the top level README.md as well.

Reply:
good point, removed it